website/src/theme/sections/features/features.js

```diff
@@ -10,57 +10,37 @@ export default function Features() {
       icon: 'red',
       title: 'Custom Block Editor blocks',
-      desc: 'Easily use our pre-made 100% customizable existing Block Editor blocks & components or create your own. We focus on dynamic blocks for developers.',
+      desc: 'Build a dynamic block faster than you can say Gutenberg. With automatic block registration and top-notch attribute handling, you\'ll be shipping blocks in record time.',
     },
     {
       icon: 'purple',
-      title: 'OOP',
-      desc: 'All code is written following the latest object-oriented programming paradigms (abstract classes, interfaces, dependency injection, namespacing).',
-    },
-    {
-      icon: 'yellow',
-      title: 'WP-CLI',
-      desc: 'Build your application with all-ready pre-made features that you can access with a single command using the terminal.',
-    },
-    {
-      icon: 'green',
-      title: 'Webpack',
-      desc: 'Modern assets & building process that you can customize.',
+      title: 'Sustainable, like code should be',
+      desc: 'All our code follows battle-tested programming practices. OOP? Check. Unit tests? Check. Code docblocks, linters for coding standards and great documentation? Triple check.',
     },
     {
       icon: 'red',
-      title: 'SASS',
-      desc: 'Easy to learn CSS on steroids.',
-    },
-    {
-      icon: 'purple',
-      title: 'Autoprefixing',
-      desc: 'You don\'t have to worry if your code will work on older browsers.',
+      title: 'OOP: The good way to write PHP code',
+      desc: 'Object-oriented programming is at the core of Eightshift Development kit, making your projects cleaner and allowing you to extend and replace functionality.',
     },
     {
       icon: 'yellow',
-      title: 'JS Compiler (Babel)',
-      desc: 'Write JS using the latest features and let Webpack worry about making it work across all browsers.',
-    },
-    {
-      icon: 'red',
-      title: 'Synchronised browser testing',
-      desc: 'Automatically minifies your production assets (JS / CSS) during the build process.',
+      title: 'wp boilerplate',
+      desc: 'Include features you need, and leave out the ones you don\'t using our powerful WP-CLI tooling.',
     },
     {
       icon: 'green',
-      title: 'Minifying',
-      desc: 'Automatically minifies your production assets (JS / CSS) during the build process.',
+      title: 'First-class CSS variables',
+      desc: 'Forget about conditional classnames: build out CSS variables from attribute values and scope them to a particular block with almost no code.'
     },
     {
-      icon: 'purple',
-      title: 'PHPCS',
-      desc: 'Helps you write better code with automatic code linting for PHP, SCSS, and JS.',
+      icon: 'red',
+      title: 'A build process to love',
+      desc: 'Eightshift Development kit comes with Webpack preconfigured. Build your JavaScript and CSS assets without breaking a sweat. Vendor prefixes? Polyfills? SCSS compilation? Asset minification? All taken care of.'
```

Comment:
```suggestion
      desc: 'Eightshift Development kit comes with Webpack preconfigured, so you can start building your JavaScript and CSS assets from the start. Vendor prefixes? Polyfills? SCSS compilation? Asset minification? All taken care of!'
```
Maybe too many _without breaking a sweat_-s in one part of the page 😄